### PR TITLE
docs: update usages/events page

### PR DESCRIPTION
**Pull request by sergkudinov**

### CODE REVIEW

File: docs/content/usages/events.md [hunk 1]
```diff
@@ -4,37 +4,72 @@ sort: 6
 
 # Events API
 
-A Nikita session extends the [native Node.js Events API](https://nodejs.org/api/events.html). It provides a facility to listen to internal notification and know the current state of the program.
+A Nikita session extends the [native Node.js Events API](https://nodejs.org/api/events.html). It provides a facility to listen to internal notifications and know the current state of the program.
```
Comment (wdavidw) [inline docs/content/usages/events.md:7]: This is wrong, we no longer extends the Events API.

File: docs/content/usages/events.md [hunk 2]
```diff
@@ -4,37 +4,72 @@ sort: 6
...
 
 ## Usage
 
-Listening to events with the Node.js API is quite simple with the `on(event, handler)`. The "handler" argument is the function provided by the user to catch events. 
+Listening to events with the extended Node.js API is quite simple with the `tools.events.on(event, handler)` function available as an argument of the [action's handler](/current/action/handler). The `handler` argument is the function provided by the user to catch events.
```
Comment (wdavidw) [inline docs/content/usages/events.md:11]: What is `extended Node.js` ?

File: docs/content/usages/events.md [hunk 3]
```diff
@@ -4,37 +4,72 @@ sort: 6
...
 
-It is not recommended to call the `emit` function directly but instead your are encouraged to call the `nikita.log` function which will associate the "type" property with the event of the same name.
+To emit an event, it is not recommended to call the `emit` function directly but instead, you are encouraged to call the `tools.log()` function which will associate the `type` property with the event of the same name.
```
Comment (wdavidw) [inline docs/content/usages/events.md:13]: I would rephrase without the `recommaned` thing. Sth like (and improved): you can emit data but most of the time, you want to use log which validate and enrich a log object and which uses `emit` internally.

File: docs/content/usages/events.md [hunk 4]
```diff
@@ -4,37 +4,72 @@ sort: 6
...
+    console.info(`I am ${log.message}`)
+  })
+  // Emit an event
+  tools.log({
+    message: 'Nikita',
+    type: 'my_event'
+  })
+})
+// Outputs with "I am Nikita"
+```
```
Comment (wdavidw) [inline docs/content/usages/events.md:30]: This page is about event, not about logging. Log shall be covered in a separate page as we will add more to it (I am planning for `log.info` for example)

**Review by wdavidw — CHANGES_REQUESTED** [review]

File: docs/content/usages/events.md (hunk 1 above)
Comment (sergkudinov) [reply docs/content/usages/events.md:7]: I replace with "use"?
`A Nikita session uses the native Node.js Events API...`

File: docs/content/usages/events.md (hunk 2 above)
Comment (sergkudinov) [reply docs/content/usages/events.md:11]: yes, it is wrong

File: docs/content/usages/events.md (hunk 1 above)
Comment (wdavidw) [reply docs/content/usages/events.md:7]: Yes, but it is not Nikita, it is Nikita with the events plugin

Comment (sergkudinov) [reply docs/content/usages/events.md:7]: you agree with this?
`Nikita is plugged with the functionality which uses the native Node.js Events API. It provides a facility to listen to internal notifications and know the current state of the Nikita session.`

File: docs/content/usages/events.md (hunk 4 above)
Comment (sergkudinov) [reply docs/content/usages/events.md]: Moved

File: docs/content/usages/events.md (hunk 3 above)
Comment (sergkudinov) [reply docs/content/usages/events.md]: Done

**Review by wdavidw — CHANGES_REQUESTED** [review]
Rephrase the wording

**Comment by sergkudinov** [comment]
@wdavidw ready

**Review by wdavidw — APPROVED** [review]
